app/views/content/what-is-qts.md

```diff
@@ -26,26 +31,43 @@ keywords:
   - State-school
 ---
 
-Qualified teacher status (QTS) is what you need to teach in most primary and secondary schools in England.
+You need qualified teacher status (QTS) to teach in maintained primary, secondary and special schools in England. These are schools funded by local authorities.
 
-It is a confirmation that you have the right training and meet the standards required to be a teacher.
+Some schools can employ teachers without QTS (for example, [private schools](https://www.gov.uk/types-of-school/private-schools) and [academy schools](https://www.gov.uk/types-of-school/academies)) but many still use it to assess the quality of candidates.
 
-QTS is a teaching standard that is respected across the world. Having QTS means you can move around the country and teach in any state school.
+You do not need QTS to teach in [further education](/become-a-further-education-teacher) or work in [early years education](/early-years-teaching-training).
 
-## How to get QTS
+$international-content$
 
-All postgraduate teacher training courses in England lead to QTS. You can see the different types of courses available by [searching for postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+## Get QTS through teacher training
```

Comment:
@EllieNodder Is this section slightly confusing with three different links? Particularly the first and third link, which sound quite similar:

* Find out more about teacher training
* postgraduate certificate in education (PGCE)
* Find out how to get QTS through teacher training

I know that the destination pages have different content but I found it a bit confusing - I'm not sure as a user I would know quite which one to pick

Reply:
@gemmadallmandfe good point. How about we ditch the first teacher training link and put that in the related content section instead, replacing the blog (bottom link)?

The main CTA for that section should be the final how to get QTS through teacher training link (and there's a link on that page to the other one anyway)